### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by louy755**

### CODE REVIEW

**Comment by louy755** [comment]
@ssharba

File: app/models/line_item.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class LineItem < ApplicationRecord
+  belongs_to :good
```
Comment (ssharba) [inline app/models/line_item.rb]: so now we have goods and products ?

Comment (louy755) [reply app/models/line_item.rb]: No I used good model and delete product model

File: app/assets/stylesheets/products.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the products controller here.
```
Comment (ssharba) [inline app/assets/stylesheets/products.scss]: remove this file

Comment (louy755) [reply app/assets/stylesheets/products.scss]: done!

**Pull request merged**